### PR TITLE
feat(curriculum): make logical OR step more concise 

### LINKED ISSUE #49484: Basic JS steps need updates to facilitate translation
### Describe the Issue

Similar to issue #49408 
Please also update another step that has the same issue and should get a similar fix:
https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-and-operator

The sentence:
"will only return Yes if num is greater than 5 and less than 10. The same logic can be written as:"
should be changed to:
"This will only..."
(adding This is enough to fix it as a reference to the previous code block)

And similarly for this step as well:
https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-or-operator

### Affected Page

Multiple

### Your code

```

```

### Expected behavior

As described above

### Screenshots

_No response_

### System

N/A

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by azr-arch**
remove grammatical mistakes stated in issue #49484

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #49484

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-or-operator.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ if (num < 5) {
 return "Yes";
 ```
 
-will return `Yes` only if `num` is between `5` and `10` (5 and 10 included). The same logic can be written as:
+This will return `Yes` only if `num` is between `5` and `10` (5 and 10 included). The same logic can be written as:
```
Comment (jeremylt) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-or-operator.md:28]: Similar to the issue, I'd replace this with
```suggestion
This code will return `Yes` if `num` is between `5` and `10` (5 and 10 included). The same logic can be written with the logical or operator.
```

and I would change the sentence on line 16 as well.

**Comment by Sboonny** [comment]
can you change line 16 as mentioned by jeremy here https://github.com/freeCodeCamp/freeCodeCamp/pull/49494#discussion_r1117316609

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-or-operator.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ if (num < 5) {
 return "Yes";
 ```
 
-will return `Yes` only if `num` is between `5` and `10` (5 and 10 included). The same logic can be written as:
+This code will return `Yes` if `num` is between `5` and `10` (5 and 10 included). The same logic can be written with the <dfn>logical or</dfn> operator.
```
Comment (Sboonny) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-or-operator.md:28]:
```suggestion
This code will return `Yes` if `num` is between `5` and `10` (5 and 10 included). The same logic can be written with the <dfn>logical OR</dfn> operator.
```

Comment (jeremylt) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-or-operator.md:28]:
```suggestion
This code will return `Yes` if `num` is between `5` and `10` (`5` and `10` included). The same logic can be written with the <dfn>logical or</dfn> operator.
```

Don't forget to update line 16

**Comment by jeremylt** [comment]
You still need to update line 16

**Pull request closed without merging**